<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAstroVolumeIOOptionsWidget</class>
 <widget class="qSlicerWidget" name="qSlicerAstroVolumeIOOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Options</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CenteredCheckBox">
     <property name="text">
      <string>Centered</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="LabelMapCheckBox">
     <property name="text">
      <string>LabelMap</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="SingleFileCheckBox">
     <property name="text">
      <string>Single File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
